services-js/access-boston/src/pages/change-password.tsx

```diff
@@ -176,15 +197,35 @@ export default class ChangePasswordPage extends React.Component<Props, State> {
 
               <div className={MAIN_CLASS}>
                 <div className="b b-c b-c--hsm">
-                  <SectionHeader title="Change Password" />
-
-                  <Formik
-                    initialValues={initialValues}
-                    validationSchema={changePasswordSchema}
-                    onSubmit={this.handleSubmit}
-                    render={this.renderForm}
+                  <SectionHeader
+                    title={
+                      setNewPassword ? 'Make a New Password' : 'Change Password'
+                    }
                   />
+
+                  {setNewPassword && (
+                    <>
+                      <p className="t--info m-t500 m-v300">
+                        You’ll need to create a new, secure password to set up
+                        your Access Boston account.
+                      </p>
+
+                      <p className="t--info m-v300">
+                        This password will be good for a{' '}
```

Comment:
Why do we need to explicitly add a space here?

Reply:
JSX collapses whitespace before HTML tags.